app/controllers/exercises_controller.rb

```diff
@@ -30,7 +30,9 @@ def start!
   end
 
   def set_default_content
+    @file_content = @exercise.files_content_for(current_user).except('content')
     @current_content = @exercise.current_content_for(current_user)
+    @files = @file_content.keys
```

Comment:
This attribute does not reference files. `@file_names` or `@filenames` perhaps?

Reply:
done